modules/mobile/spec/request/gender_identity_request_spec.rb

```diff
@@ -23,12 +23,19 @@
 
   describe 'GET /mobile/v0/gender_identity/edit' do
     context 'requested' do
-      it 'returns a list of valid ids' do
+      before do
         get('/mobile/v0/user/gender_identity/edit', headers: iam_headers)
+      end
 
+      it 'returns a list of valid ids' do
         json = json_body_for(response)['attributes']['options']
         expect(json).to eq(VAProfile::Models::GenderIdentity::OPTIONS.transform_keys(&:downcase))
       end
+
+      it 'returns a list in correct order' do
+        codes = response.parsed_body.dig('data', 'attributes', 'options').pluck(0)
```

Comment:
This works but I'd recommend going with the more obvious (to a reader):
`response.parsed_body.dig('data', 'attributes', 'options').keys`